docs/environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
 - jupyter_client
 - ipykernel
 - pip:
-  - sphinx<1.4
+  - sphinx
```

Comment:
OMG. In fact we should move sphinx above the pip section.

Reply:
Thanks, done